catroid/src/main/java/org/catrobat/catroid/content/actions/ArduinoSendDigitalValueAction.java

```diff
@@ -66,12 +66,7 @@ protected void begin() {
 					interpretationException);
 		}
 
-		if ((!restart)
-				&& (pinNumberInterpretation == 2)
-				|| (pinNumberInterpretation == 4)
-				|| (pinNumberInterpretation == 8)
-				|| (pinNumberInterpretation == 12)
-				|| (pinNumberInterpretation == 13)) {
+		if (!restart) {
 			this.pin = pinNumberInterpretation;
 			this.value = pinValueInterpretation;
 		}
```

Comment:
> Is the method ArduinoListener.setPortValue even needed?
You're right - it's actually unnecessary (in the current code). 
However it makes sense to keep it - as CAT-2142 [#1953](https://github.com/Catrobat/Catroid/pull/1953) introduces a port variable (it renames portValue to pinValue). By removing the setter those two arrays would easily get out of sync - with no immediate problems, but some likely problems in the future if someone assumes they contain the same consistent data.